src/main/java/ch/dvbern/kibon/exchange/api/common/shared/BetreuungDTO.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (C) 2020 DV Bern AG, Switzerland
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ch.dvbern.kibon.exchange.api.common.shared;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.StringJoiner;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import ch.dvbern.kibon.exchange.api.common.institution.InstitutionDTO;
+import ch.dvbern.kibon.exchange.api.common.platzbestaetigung.BetreuungZeitabschnittDTO;
+import ch.dvbern.kibon.exchange.api.common.verfuegung.BetreuungsAngebot;
+
+public class BetreuungDTO implements Serializable {
+
+	private static final long serialVersionUID = -806547086175525904L;
+
+	/**
+	 * Older (maybe unused) fields of the DTO have been made Nullable for now.
+	 * Can be removed, if not in use
+	 */
+	@Nullable
+	private BetreuungsAngebot betreuungsArt;
+
+	@Nullable
+	private @Valid InstitutionDTO institution;
+
+	@Nonnull
+	private @Size(min = 1) @NotNull String referenznummer;
+
+	private boolean ausserordentlicherBetreuungsaufwand;
+
+	@Nullable
+	private List<Long> gemeindeBfsNrs;
```

Comment:
A list does not really make sense here. If you want to support multipe Gemeinde, then this information should move to BetreuungZeitabschnittDTO.

Since kiBon can only accept 1 Gemeinde per period, I would simply change the List to a single value.